NEWS

```diff
@@ -21,8 +21,59 @@ example, a bug might be fixed in the master, and then moved to the
 current release as well as the "stable" bug fix release branch.
 
 
-Master (not on release branches yet)
+3.0.0 -- TBD
 ------------------------------------
+**** NOTE: This release implements the complete PMIX v3.0 Standard
+**** and therefore includes a number of new APIs and features. These
+**** can be tracked by their RFC's on the community website:
+**** https://pmix.org/pmix-standard.
+- Added blocking forms of several existing APIs:
+    - PMIx_Log
+    - PMIx_Allocation_request
+    - PMIx_Job_control
+    - PMIx_Process_monitor
+- Added support for getting/validating security credentials
+    - PMIx_Get_credential, PMIx_Validate_credential
+- Extended support for debuggers/tools
+    - Added IO forwarding support allowing tools to request
+      forwarding of output from specific application procs,
+      and to forward their input to specified target procs
+    - Extended tool attributes to support synchronization
+      during startup of applications. This includes the
+      ability to modify an application's environment
+      (including support for LD_PRELOAD) and define an
+      alternate fork/exec agent
+    - Added ability for a tool to switch server connections
+      so it can first connect to a system-level server to
+      launch a starter program, and then reconnect to that
+      starter for debugging purposes
+- Extended network support to collect network inventory by
```

Comment:
Did this one get merged - the RFC (https://github.com/pmix/RFCs/pull/30) is still open - should it be closed?